### PR TITLE
Exec Doc 2024-10-31

### PR DESCRIPTION
**Pull request by blimpa**
@VoteWizard , @CivicSage3

### CODE REVIEW

File: governance/votes/Executive vote - October 31, 2024.md
Comment (boet1) [inline governance/votes/Executive vote - October 31, 2024.md:65]:
```suggestion
  - `l1BridgeImpCLKey`: BASE_TOKEN_BRIDGE_IMP (chainlog key for L1TokenBridgeImp)
```

File: governance/votes/Executive vote - October 31, 2024.md
Comment (boet1) [inline governance/votes/Executive vote - October 31, 2024.md:45]:
```suggestion
  - `bridgeImp`: [0xaeFd31c2e593Dc971f9Cb42cBbD5d4AD7F1970b6](https://etherscan.io/address/0xaeFd31c2e593Dc971f9Cb42cBbD5d4AD7F1970b6)
```

File: governance/votes/Executive vote - October 31, 2024.md
Comment (VoteWizard) [inline governance/votes/Executive vote - October 31, 2024.md:64]:
```suggestion
  - `l1BridgeCLKey`: BASE_TOKEN_BRIDGE (chainlog key for L1TokenBridge)
```

File: governance/votes/Executive vote - October 31, 2024.md
Comment (MakerJanSky) [inline governance/votes/Executive vote - October 31, 2024.md:22]: Perhaps add a link or more clearly indicate these are for the OSM.

File: governance/votes/Executive vote - October 31, 2024.md
Comment (MakerJanSky) [inline governance/votes/Executive vote - October 31, 2024.md:106]: I am not a fan of this formatting, it reads awkwardly. Additionally, I think more resources on the Medianizer should be added.

**Review by MakerJanSky — COMMENTED** [review]
Nice work, all. We're still awaiting the final approvals which will need to be added. My other suggestions are non-blocking but should be considered.

**Review by 0xldr — CHANGES_REQUESTED** [review]

**Review by MakerJanSky — APPROVED** [review]
Nice work, all.

**Comment by VoteWizard** [comment]
# Spell Sheet Creation

## Pre-Meeting Tasks

- [x] Reach out to Ecosystem Actors about possible Spell Contents.
- [x] Inform relevant parties about deadlines (see [Spell Coordination Schedule](https://github.com/makerdao/pe-checklists/blob/master/spell/spell-crafter-mainnet-workflow.md#spell-coordination-schedule)).
- [x] Review recent [polls](https://vote.makerdao.com/polling) for to-do items.
- [x] Review the latest [forum posts](https://forum.makerdao.com/latest) for requested items.

## GovOps Meeting Tasks (Tuesdays, one week before Spell release)

- [x] Obtain roles for this week:
    - [x] Spell Crafter
    - [x] Reviewer (Dewiz)
    - [x] Reviewer (Sidestream)
- [x] Review Requested Contents from Ecosystem Actors.
- [x] Obtain consensus on Spell Items.
- [x] Confirm Repeating Checklist Items:
    - [x] Office Hours (Yes/No)
    - [x] Global Line Modifier (Yes/No)
    - [x] Order of Operations Issues (Yes/No)

## Sheet Creation (Tuesdays, one week before Spell release)

- [x] Duplicate "Executive Contents (Template)" in [Spell Sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?usp=sharing).
- [x] Add a new column with the spell date in the "Spell Progress Tracking" sheet and ensure the progress is kept up to date.
- [x] Rename duplicate tab for expected spell date (in yyyy-mm-dd format).
- [x] Add role and recurring items info from GovOps Meeting.
- [x] Add Spell Items as formatted in the sheet. 
- [x] Complete and confirm checksums for DAI and MKR transfers.
- [x] SubDAO Spell Crafter delivered the SubDAO spell address and spell PR in Discord with no differences in the code.

## Obtain Confirmations for listed items 

- [x] Reach out to relevant Ecosystem Actors for item-specific confirmations.
- [x] Correct any discrepancies.
- [x] Confirm items for the `Governance Confirm` column.

**Comment by blimpa** [comment]
# Executive Creation

## Creation

- [x] Sync Community Git
- [x] Ensure you are working from the latest master branch
- [x] Read Associated [Executive Contents sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?pli=1#gid=1593813984)
- [x] Copy the executive template
- [x] Update filename
- [x] Update dates - exact time is not required and should be left at midnight.

- [x] Ensure the title refers to the most impactful/important actions included in the proposal.
- [x] Ensure that the summary refers to all actions included in the proposal.

- [x] Ensure that the publisher (entity publishing the executive on the voting portal) of the executive is listed in the preamble (at the entity level).
- [x] Ensure that the writers of the executive code are listed in the preamble (at the entity level).
- [x] Ensure that the reviewers of the executive code are listed in the preamble (at the entity level).

- [x] Ensure that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
- [x] Ensure that it is clearly stated which items are affected by the GSM Delay.
- [x] Ensure that the GSM Delay period is clearly communicated.
- [x] Ensure that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.

## Items

- [x] Add each item to the proposal details section
- [x] Order the items by judged impact/importance. If different from the executive contents, reorder the executive contents sheet so that they match.

- [x] For each item
  - [x] Ensure the executive summary contains a high-level overview of every action contained in the proposal.
  - [x] Ensure that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
  - [x] Ensure that each item in the copy contains a link to the reasoning for its inclusion.
  - [x] Ensure that each item in the copy contains a link to the authorization for its inclusion.
  - [x] Ensure that each item in the copy contains any links required for voters to determine the context surrounding its inclusion.
  - [x] Ensure that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
  - [x] Ensure that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
  - [x] Ensure that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
  - [x] Ensure that parties implicated in the proposal outcome related to this item are aware of the wording used.

## Self-Review

- [x] Do any '$' parameter signifiers remain? (spell address is unknown at this point)
- [x] Do the executive title, summary, and executive summary accurately reflect executive copy content?
- [x] Read the executive copy in full, with the above checklists and general readability in mind. Make any required changes.
- [x] Run the copy through a spelling and grammar checker - do not blindly accept suggested changes.
- [x] Inform the reviewer that the executive copy is ready to review
- [x] Commit changes, and create a PR to the Community GitHub repository.

## Submission

- [] Merge approved executive copy PR.
- [] Inform technical teams that the copy is ready and provide the link.
- [] Once technical teams have provided a spell address, add it to the executive copy.
- [] Update the proposals.json to set the active proposal on the portal.
- [] Refresh portal cache (DUX/Jetstream can provide password)
- [] Inform the community of the new executive proposal.

**Comment by VoteWizard** [comment]
### Hash of the Document Before Merging

Command:
`cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/820339cf860e625924811f181062851d7e25b610/governance/votes/Executive%20vote%20-%20October%2031%2C%202024.md' -q -O - 2>/dev/null)"`

Hash:
`0x6407f9203bf4f816cc353ebc95463d917e77ccb701f2e85945dcf91274b628ed`

The hash matches with the outcome of this online tool:
https://emn178.github.io/online-tools/keccak_256.html

**Comment by CivicSage3** [comment]
# Executive Document Review

## Content
- [x] Check the title refers to the most impactful/important actions included in the proposal.
- [x] Check that the summary refers to all actions included in the proposal.
- [x] Check that the publisher (entity publishing the executive on the voting portal) of the executive is listed in the preamble (at the entity level).
- [x] Check that the writers of the executive code are listed in the preamble (at the entity level).
- [x] Check that the reviewers of the executive code are listed in the preamble (at the entity level).
- [x] Check that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
- [x] Check that it is clearly stated which items are affected by the GSM Delay.
- [x] Check that the GSM Delay period is communicated.
- [x] Check that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.

## Items

- [x] Check each item is present in the proposal details section
- [x] Check items are ordered by judged impact/importance
- [x] For each item
	- [x] Check the executive summary contains a high-level overview of the item.
	- [x] Check that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
	- [x] Check that the copy contains a link to the reasoning for its inclusion.
	- [x] Check that the copy contains a link to the authorization for its inclusion.
	- [x] Check that the copy contains any links required for voters to determine the context surrounding its inclusion.
	- [x] Check that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
	- [x] Check that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
	- [x] Check that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
	- [x] Check that parties implicated in the proposal outcome related to this item are aware of the wording used.

## Final Checks

- [x] Read the executive copy in full, with the above checklists and general readability in mind. Raise any issues with the author.

**Pull request merged**
